### PR TITLE
OCPBUGS-1153: Add etcd operator roles and clusterrole 

### PR DESCRIPTION
**Pull request by tjungblu**
cc @Elbehery

### CODE REVIEW

**Comment by tjungblu** [comment]
a little further than before, it's still not picking up the right permissions though:

> W0307 11:35:56.170919       1 builder.go:230] unable to get owner reference (falling back to namespace): pods "etcd-operator-8dd67b5f-fgg98" is forbidden: User "system:serviceaccount:openshift-etcd-operator:etcd-operator" cannot get resource "pods" in API group "" in the namespace "openshift-etcd-operator"


> I0307 11:35:56.718457       1 server.go:93] Error checking for connectivity to apiserver (GET lease openshift-etcd-operator/openshift-cluster-etcd-operator-lock): leases.coordination.k8s.io "openshift-cluster-etcd-operator-lock" is forbidden: User "system:serviceaccount:openshift-etcd-operator:etcd-operator" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace "openshift-etcd-operator"

which are both included in the cluster role and in the operator namespace role :/

File: manifests/0000_20_etcd-operator_04_clusterrolebinding.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: system:openshift:operator:etcd-operator
```
Comment (stlaz) [inline manifests/0000_20_etcd-operator_04_clusterrolebinding.yaml:4]: that name was correct

File: manifests/0000_20_etcd-operator_04_clusterrole.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: etcd-operator-cluster-role
```
Comment (stlaz) [inline manifests/0000_20_etcd-operator_04_clusterrole.yaml:4]: `system:openshift:etcd-operator-cluster-role`

**Review by stlaz — CHANGES_REQUESTED** [review]

**Comment by Elbehery** [comment]
well done @tjungblu  💯 👍🏽

**Comment by Elbehery** [comment]
@tjungblu shall we close this, as we `wont do` ?

**Comment by tjungblu** [comment]
yep

/close

**Pull request closed without merging**